plugins/inputs/mqtt_consumer/mqtt_consumer_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/eclipse/paho.mqtt.golang"
+	mqtt "github.com/eclipse/paho.mqtt.golang"
```

Comment:
Could you please elaborate on why this is necessary!?

Reply:
this was automatically added by the formatting library, (I think goreturns, in this case). It doesn't like the implicit import of the mqtt package via a path that doesn't match the package name.

Reply:
Ok so this was my "TIL". :-) Thanks for the explanation!